tests/rules/experimental/rbe/BUILD

```diff
@@ -12,9 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("//rules/experimental/rbe:exec_properties.bzl", "create_exec_properties_dict")
+load("@exec_properties//:exec_propert_sets.bzl", "DOCKER_PRIVILEGED", "NETWORK_ON")
```

Comment:
we're loosing test coverage of create_exec_properties_dict. Is it necessary to keep it? Is now create_exec_properties_dict an implementation detail? If so, can we document it as so?

Reply:
It's not exactly an implementation detail.
We are still calling it, but now we are doing so from the WORKSPACE file.
For example, it was used in the creation of exec_properties_with_override and network_on_exec_properties.

Reply:
If its still being used, should we still have some tests that exercise it independently of the new rule? It seems its only now being tested indirectly via the repo rules.

Reply:
I'm inclined to say no.
Calling create_exec_properties_dict() from a BUILD file, while not impossible, is an anti-pattern.
I'm actually interested in putting in place some kind of linter check (maybe in buildifier?) that would prevent that anti-pattern, or maybe just give out a warning.

Reply:
I'm not referring to whether customers should call this from a BUILD file or not. I'm referring to how we are testing the rule from within this repo. Prior to this PR, we had some form of unit tests that directly checked the create_exec_properties_dict rule, with this PR we do not, we have tests that indirectly depend on create_exec_properties_dict. Lets re add them, even if users should not ever call create_exec_properties_dict from a BUILD file

Reply:
But then if and when we add this linter check to buildifier, this particular build file will fail the lint test.

Reply:
We can always turn off that check for this particular BUILD file

Reply:
Following our offline discussion, I agree that we *could* turn off the check for this particular BUILD file.
The question we need to answer, I think, is would testing create_exec_properties_dict() directly from the BUILD file (despite being an anti-pattern) add extra test coverage that we don't otherwise have.
If the answer is yes, then I am happy to make a linter exception here to improve code coverage. But I don't think it does add more coverage.